libs/map/src/components/legend-item/vl-map-legend-item.ts

```diff
@@ -2,20 +2,25 @@ import { html, PropertyDeclarations, TemplateResult, CSSResult } from 'lit';
 import { customElement } from 'lit/decorators.js';
 import { BaseLitElement } from '@domg-wc/common-utilities';
 import legendStyles from '../legend/vl-map-legend.uig-css';
+import legendItemStyles from './vl-map-legend-item.uig-css';
 
 @customElement('vl-map-legend-item')
 export class VlMapLegendItem extends BaseLitElement {
     icon = null;
     label = '';
     layer = '';
+    iconText = '';
+    iconTextColor = '';
 
     static get styles(): (CSSResult | CSSResult[])[] {
-        return legendStyles;
+        return [legendStyles, legendItemStyles];
     }
 
     static get properties(): PropertyDeclarations {
         return {
             layer: { type: String, attribute: 'data-vl-layer', reflect: true },
+            iconText: { type: String, attribute: 'data-vl-icon-text', reflect: true },
```

Comment:
`reflect: true` mag je weglaten, als we de waarde van het attribuut intern nooit veranderen heeft dit geen nut.
Je mag dit ook verwijderen voor het `layer` attribuut.